### PR TITLE
Update Readme Files for FoodAdvisor

### PR DESCRIPTION
**Pull request by davidkartuzinski**
The Readme files were updated to reflect changes in the Beta version of Strapi.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 ## Client
 
-Run `cd client && npm install && npm start`
+Run `cd client && yarn start`
```
Comment (lauriejim) [inline README.md]: You remove the `npm install` command.
You still need to run `yarn`

File: api/README.md [hunk 2]
```diff
@@ -4,21 +4,17 @@
 
 In the project directory, you can run:
 
-### `npm install`
+### `yarn install`
```
Comment (lauriejim) [inline api/README.md]: To be consistent with the readme file we can use `yarn` instead of `yarn install`

File: api/README.md [hunk 3]
```diff
@@ -4,21 +4,17 @@
...
 
-`npm install` installs all dependencies to make this project work.
+`yarn install` installs all dependencies to make this project work.
 
-### `npm run setup --plugins`
+### `yarn run seed`
```
Comment (lauriejim) [inline api/README.md]: Here we can just use `yarn seed` - we can remove `run` from the command.
This comment is applicable in other commands.

File: README.md (hunk 1 above)
Comment (davidkartuzinski) [reply README.md]: This is back. Thank you

File: api/README.md (hunk 2 above)
Comment (davidkartuzinski) [reply api/README.md]: I don't understand.  Everything is now consistent across the entry Readme and the API and client readme.  @lauriejim

File: api/README.md (hunk 3 above)
Comment (davidkartuzinski) [reply api/README.md]: Done.  I adjusted the remaining commands.

**Pull request merged**